### ISSUE TITLE
Inconsistent delimiters between code and non-code content areas

### ISSUE THREAD

**Issue by DaveParillo** [RunestoneInteractive/RunestoneComponents#616]
There doesn't seem to be a clear reason why the separator between the question and code in a parsons problem would be `----` and in an activecode directive would be `~~~~`.

It would be easier for authors to remember these kinds of details if they were more consistent.

Not that this isn't fixable if an author forgets, but this just increases the cognitive burden.

**Comment by bnmnetp** [comment]
Indeed, if only the maintainer were better about checking details like that from multiple developers submitting PRs.  😬 

The trouble with just fixing it right now is that there is so much material that is already out there we would need to do something to maintain backward compatibility, which would not be that hard I guess if activecode checked for 4 or 5 of `~` or `=` that would keep it backward compatible and I can't imagine it would break anything.
